### DISCUSSION TITLE
Error 404

### DISCUSSION THREAD

**Discussion by leandroheffes24** [vercel/community#5573] [Help]
### Summary

I uploaded a github project made with node.js and the 404 error appears with the following id: "gru1::rvj5j-1706026506378-9f21211d972a"

### Steps to Reproduce

![error](https://github.com/vercel/community/assets/136512218/e722cc67-596a-41c6-8095-b4e828b6ebd6)

**Comment by silentk0i** [comment]
Can you provide the vercel build logs?

**Reply by leandroheffes24** [reply]
<html>
<body>

Running build in Washington, D.C., USA (East) – iad1
--
16:38:14.437 | Cloning github.com/leandroheffes24/sw-asistencia-virtual (Branch: main, Commit: 92b61c9)
16:38:14.824 | Previous build cache not available
16:38:15.162 | Cloning completed: 724.906ms
16:38:15.374 | Running "vercel build"
16:38:15.874 | Vercel CLI 33.2.0
16:38:16.046 | Build Completed in /vercel/output [12ms]
16:38:16.093 | Deploying outputs...
16:38:20.647 |  
16:38:20.883 | Deployment completed
16:38:21.426 | Uploading build cache [29.00 B]...
16:38:21.808 | Build cache uploaded: 381.829ms


</body>
</html>

**Comment by GameMakerPremium** [comment]
I'm getting the same error. But I'm not using node.js. Is this a common error?
![image](https://github.com/vercel/community/assets/50556150/6f4b523a-2d72-4839-8cb7-358e0c4bc80b)
![image](https://github.com/vercel/community/assets/50556150/1b2cf83d-cc06-4d9c-87a8-34b41d790c47)
